### ISSUE TITLE
Does not count link clicks.

### ISSUE THREAD

**Issue by jackkweyunga** [kalokola/ellipsis_django_task#1]
Shortie does not seem to count link clicks.

**Comment by kalokola** [comment]
Yes, it doesn't count the number of clicks, in the assignment nlotumiwa
sikuambiwa kuhesabu clicks.

**Comment by zipa-tech** [comment]
Yes it doesn't,  in the assignment provided to me I wasn't told to count the number of clicks per link. I can easily add it.

**Comment by kalokola** [comment]
Yes it doesn't, in the assignment provided to me I wasn't told to count the number of clicks per link. I can easily add it.

**Comment by kalokola** [comment]
Thanks brother, I was doing an interview somewhere,I couldn't design out of
scope.  I just placed the repository there for them to clone. Regards.
